packages/react-sdk/docusaurus/docs/React/06-ui-cookbook/14-layout-customization.mdx

```diff
@@ -0,0 +1,182 @@
+---
+id: layout-customization
+title: Customization of default call layouts and ParticipantView component
+---
+
+import ParticipantViewUIImage from '../assets/ui-cookbook/14-layout-customization/participant-view-ui.png';
+
+## Introduction
+
+In this guide you'll learn how to customize default call layouts and how to apply custom video placeholder and UI component to `ParticipantView` component. We recommend using the `ParticipantView` component provided by the SDK because it handles rendering your video, manages video quality updates, subscriptions, and visibility state when the element is not in the visible viewport. By following this guide, you'll be able to tailor the call layouts and visual elements according to your specific needs while leveraging the capabilities provided by the `ParticipantView` component.
+
+### Purpose of the `ParticipantViewUI` and `VideoPlaceholder` templates
+
+`ParticipantView` comes by default with certain participant information displayed in the "box" - such as mute states of either video or audio, quality indicator, participant's name, reactions etc. which are displayed over the video element positioned absolutely (see picture). By utilizing the `ParticipantViewUI` prop, you can extend the `ParticipantView` component with custom UI elements specific to each participant. This allows you to tailor the visual representation of the participant based on your application's requirements and design.
+
+<img src={ParticipantViewUIImage} alt="ParticipantViewUI elements" />
+
+The `VideoPlaceholder` prop allows you to replace the default placeholder displayed over the video element when the video is not playing. This placeholder is positioned absolutely over the video element\*. It provides you with the flexibility to customize the placeholder's appearance and content according to your needs.
+
+:::caution
+\*your custom `VideoPlaceholder` component should be able to accept `style` prop coming from `ParticipantView` to allow for absolute positioning
+:::
+
+## Customization in SDK provided layout components (preferred way)
+
+We highly recommend using layout components provided by the SDK as modification of the `ParticipantViewUI` in these call layouts is a possiblity. You can follow [this guide](./13-custom-call-layout.mdx) to build your own call layout.
+
+### `PaginatedGridLayout`
```

Comment:
Pretty sure it's planned to have that there but a short explanation for it would be awesome. :)